ddl/ddl_api.go

```diff
@@ -2594,10 +2595,11 @@ func (d *ddl) preSplitAndScatter(ctx sessionctx.Context, tbInfo *model.TableInfo
 }
 
 func (d *ddl) FlashbackCluster(ctx sessionctx.Context, flashbackTS uint64) error {
+	logutil.BgLogger().Info("[ddl] get flashback cluster job", zap.String("flashbackTS", oracle.GetTimeFromTS(flashbackTS).String()))
 	job := &model.Job{
 		Type:       model.ActionFlashbackCluster,
 		BinlogInfo: &model.HistoryInfo{},
-		Args:       []interface{}{flashbackTS, map[string]interface{}{}},
+		Args:       []interface{}{flashbackTS, tablecodec.EncodeTablePrefix(0), map[string]interface{}{}},
```

Comment:
If `tablecodec.EncodeTablePrefix(0)` is fixed, could we put it to `onFlashbackCluster` directly? If can't do it, please add a comment.

Reply:
Ok, I will delete it in this PR and add back in next PR.